nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -1012,6 +1012,30 @@ protected static void configureSslContextFactory(SslContextFactory.Server contex
         contextFactory.setIncludeProtocols(TlsConfiguration.getCurrentSupportedTlsProtocolVersions());
         contextFactory.setExcludeProtocols("TLS", "TLSv1", "TLSv1.1", "SSL", "SSLv2", "SSLv2Hello", "SSLv3");
 
+        // on configuration, replace default application cipher suites with those configured
+        final String includeCipherSuitesProps = props.getProperty(NiFiProperties.WEB_HTTPS_INCLUDE_CIPHERSUITES);
+        if (!StringUtils.isEmpty(includeCipherSuitesProps)) {
+            logger.info(String.format("NiFiProperties.%s = %s",
+                    NiFiProperties.WEB_HTTPS_INCLUDE_CIPHERSUITES, includeCipherSuitesProps));
+            final String[] includeCipherSuitesRuntime = contextFactory.getIncludeCipherSuites();
+            final String[] includeCipherSuites = includeCipherSuitesProps.split(",\\s*");
+            logger.info(String.format("Replacing include cipher suites with configuration; FROM [%s] TO [%s].",
+                    ((includeCipherSuitesRuntime == null) ? null : Arrays.asList(includeCipherSuitesRuntime)),
+                    String.join(", ", Arrays.asList(includeCipherSuites))));
+            contextFactory.setIncludeCipherSuites(includeCipherSuites);
+        }
+        final String excludeCipherSuitesProps = props.getProperty(NiFiProperties.WEB_HTTPS_EXCLUDE_CIPHERSUITES);
+        if (!StringUtils.isEmpty(excludeCipherSuitesProps)) {
+            logger.info(String.format("NiFiProperties.%s = %s",
+                    NiFiProperties.WEB_HTTPS_EXCLUDE_CIPHERSUITES, excludeCipherSuitesProps));
+            final String[] excludeCipherSuitesRuntime = contextFactory.getExcludeCipherSuites();
+            final String[] excludeCipherSuites = excludeCipherSuitesProps.split(",\\s*");
```

Comment:
Though it would only be used twice, it might make sense to extract the splitting call into a method, or at least make ",\\s*" a constant.

Reply:
Sure.  Good opportunity here for extracting a method, and definitely want to make the string literal into a constant.